### PR TITLE
Feature/main alcs 910

### PR DESCRIPTION
**Pull request by lstod**
Adding OATS to prod SchemaSpy

### CODE REVIEW

File: ora2pg/data/ddl_18_2_prod.sql [hunk 1]
```diff
@@ -0,0 +1,4162 @@
+-- Generated by Ora2Pg, the Oracle database Schema converter, version 18.2
```
Comment (dhaselhan) [inline ora2pg/data/ddl_18_2_prod.sql:1]: Did you mean to add this file? Seems unrelated to Schemaspy.

File: .github/workflows/schemaspy.yml [hunk 2]
```diff
@@ -34,14 +34,18 @@ jobs:
           npm install
           npm run alcs:migration:run
         working-directory: services
+      - name: Import OATS
```
Comment (dhaselhan) [inline .github/workflows/schemaspy.yml:37]: Looks like this will run every time we generate schemaspy, is that necessary?

File: ora2pg/data/ddl_18_2_prod.sql (hunk 1 above)
Comment (dhaselhan) [inline ora2pg/data/ddl_18_2_prod.sql:1]: Why is this called ddl_18_2_prod? Should this have a more relatable name like oats_schemadump_ddl.sql or something?

File: .github/workflows/schemaspy.yml (hunk 2 above)
Comment (lstod) [reply .github/workflows/schemaspy.yml]: removed!

**Review by dhaselhan — APPROVED** [review]
LGTM 👍
